packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -154,6 +151,19 @@ export const BASIC_FALSY_TESTS = {
 
 export class SharedConditionalsTest extends AbstractConditionalsTest {
 
+  renderValues(...values) {
+    let templates = [];
+    let context = {};
+
+    for (let i = 1; i <= values.length; i++) {
+      templates.push(this.templateFor({ cond: `cond${i}`, truthy: `{{t}}${i}`, falsy: `{{f}}${i}` }));
+      context[`cond${i}`] = values[i - 1];
+    }
+
+    let wrappedTemplate = this.wrapperFor(templates);
+    this.render(wrappedTemplate, assign({ t: 'T', f: 'F' }, context));
+  }
```

Comment:
Isn't this implementation specific to the syntax version? (because it uses `{{t}}...`) Shouldn't this be in `SharedSyntaxConditionalsTest`? (it might be related to me not understanding the difference between the class hierarchy)